core/src/main/java/com/linecorp/armeria/internal/common/AbstractHttp2ConnectionHandler.java

```diff
@@ -71,6 +74,9 @@
     private boolean closing;
     private boolean handlingConnectionError;
 
+    // Debug data that will be sent in the GOAWAY frame.
```

Comment:
Could use Javadoc for better IDE support:
```suggestion
    /** Debug data that will be sent in the GOAWAY frame. */
```